wrapped-bridged-usdc/src/lib.rs

```diff
@@ -0,0 +1,120 @@
+#![no_std]
+
+elrond_wasm::imports!();
+
+#[elrond_wasm::contract]
+pub trait WrappedBridgedUsdc {
```

Comment:
I know that now this is the case for USDC - but the names should reflect that we want to to the same for USDT, ETH, BTC, BNB and some others. It is ok for now - but we might do some renaming. Please add a task in jura to rename.